.goreleaser.yml

```diff
@@ -18,12 +18,7 @@ builds:
       - -X 'github.com/auth0/auth0-cli/internal/buildinfo.BuildDate={{.Date}}'
       - -X 'github.com/auth0/auth0-cli/internal/instrumentation.SentryDSN={{.Env.SENTRY_DSN}}'
 archives:
-  - replacements:
-      darwin: Darwin
-      linux: Linux
-      windows: Windows
-      386: i386
-      amd64: x86_64
+  - name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
```

Comment:
Would

```
{{ .ProjectName }}_{{ .Version }}_{{ title .Os }}_{{ if eq .Arch "amd64" }}x86_64{{ else }}{{ .Arch }}{{ end }}
```
produce the current pattern we're using?

<img width="318" alt="Screenshot 2023-11-13 at 15 09 32" src="https://github.com/auth0/auth0-cli/assets/5055789/e7e6912d-5c65-42f5-b761-b1417b56d0b8">

Reply:
Except for the `*_x86_64` portions. Those show as `*_amd64` and since we build arm, it appends `_v1`. I think the template is wrong in the latter respect: I haven't had a go at turning the `*_amd64` into `*_x86_64` yet, but open to suggestions.

Reply:
For example (ignore the SNAPSHOT portion): 
<img width="410" alt="image" src="https://github.com/auth0/auth0-cli/assets/9272/1d982e9e-5d26-4df9-a650-10eeb0728ca8">

Reply:
When run it looks like:

```
auth0-cli_v1.2.0-SNAPSHOT-60aa1ac_darwin_amd64.tar.gz
```

But, when run through GitHub releases, it will look like this:

```
auth0-cli_v1.2.0_darwin_amd64.tar.gz
```

Reply:
I managed to fix that issue.